<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutroWidget</class>
 <widget class="QWidget" name="OutroWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="tempLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Outro page under construction</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="replayButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>440</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Replay</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
